balancer-js/src/modules/pools/factory/composable-stable/composable-stable.factory.ts

```diff
@@ -20,19 +37,20 @@ export class ComposableStableFactory implements PoolFactory {
     this.wrappedNativeAsset = tokens.wrappedNativeAsset;
   }
 
-  /**
-   * Builds a transaction for a composable pool create operation.
-   * @param contractAddress - The address of the factory for composable stable pool (contract address)
-   * @param name - The name of the pool
-   * @param symbol - The symbol of the pool
-   * @param swapFee - The swapFee for the owner of the pool in string or number format(100% is "1.00" or 1, 10% is "0.1" or 0.1, 1% is "0.01" or 0.01)
-   * @param tokenAddresses - The token's addresses
-   * @param rateProviders The addresses of the rate providers for each token, ordered
-   * @param tokenRateCacheDurations the Token Rate Cache Duration of each token
-   * @param owner - The address of the owner of the pool
-   * @param amplificationParameter The amplification parameter(must be greater than 1)
-   * @param exemptFromYieldProtocolFeeFlags array containing boolean for each token exemption from yield protocol fee flags
-   * @returns a TransactionRequest object, which can be directly inserted in the transaction to create a composable stable pool
+  /***
+   * @param params
+   *  * Builds a transaction for a composable pool create operation.
+   *  * @param contractAddress The address of the factory for composable stable pool (contract address)
+   *  * @param name The name of the pool
+   *  * @param symbol The symbol of the pool
+   *  * @param swapFee The swapFee for the owner of the pool in string or number format(100% is "1.00" or 1, 10% is "0.1" or 0.1, 1% is "0.01" or 0.01)
```

Comment:
Is this comment format starting with `* * @param` auto-generated somehow? It does not follow the same pattern we have on other method descriptions.
Besides that, we're usually using `EVM scale` for inputs such as `swapFee` is there a reason to why we should use `human scale` here?

Reply:
Bruno is right here:
- Keep same pattern for docs. Mine gets auto generated when I start typing `/**`
- We should keep consistency to other methods and use EVM scale for inputs.

Reply:
I'm using /** to generate the comments too, idk why is different 🤔About the EVM scale, you're totally right, I need to change to evm scale there